internal/uvm/create.go

```diff
@@ -28,12 +41,15 @@ type UVMOptions struct {
 	LayerFolders []string // Set of folders for base layers and scratch. Ordered from top most read-only through base read-only layer, followed by scratch
 
 	// LCOW specific parameters
-	KirdPath              string // Folder in which kernel and initrd reside. Defaults to \Program Files\Linux Containers
-	KernelFile            string // Filename under KirdPath for the kernel. Defaults to bootx64.efi
-	InitrdFile            string // Filename under KirdPath for the initrd image. Defaults to initrd.img
-	KernelBootOptions     string // Additional boot options for the kernel
-	EnableGraphicsConsole bool   // If true, enable a graphics console for the utility VM
-	ConsolePipe           string // The named pipe path to use for the serial console.
+	BootFilesPath         string              // Folder in which kernel and root file system reside. Defaults to \Program Files\Linux Containers
+	KernelFile            string              // Filename under BootFilesPath for the kernel. Defaults to bootx64.efi
+	RootFSFile            string              // Filename under BootFilesPath for the UVMs root file system. Defaults are initrd.img or rootfs.vhd.
+	PreferredRootFSType   PreferredRootFSType // Controls searching for the RootFSFile if omitted.
+	KernelBootOptions     string              // Additional boot options for the kernel
+	EnableGraphicsConsole bool                // If true, enable a graphics console for the utility VM
+	ConsolePipe           string              // The named pipe path to use for the serial console.  eg \\.\pipe\vmpipe
+	VPMemDeviceCount      int32               // Number of VPMem devices. Limit at 128. If booting UVM from VHD, device 0 is taken.
+	NoSCSI                bool                // The utility VM does not have any SCSI controllers. Useful in some service VM scenarios.
```

Comment:
Same as my comment above. How about SCSICount: 0, 1, 2 etc.

Reply:
That's updated now.